core/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM python:2-alpine3.10
+FROM python:3.6-alpine3.10
```

Any reason not to use Python 3.7? It's newer and will be supported longer.
```suggestion
FROM python:3.7-alpine3.10
```

---

Not every module seems to be 3.7 compatible:
https://github.com/certbot/certbot/blob/v0.37.2/certbot-dns-ovh/setup.py#L38-L44

---

Not listing Python 3.7 support there is a mistake, likely due to this order of events:

1. PRs for those plugins were opened.
2. We added Python 3.7 support to all of our code.
3. PRs were merged.

I created a PR to fix this at https://github.com/certbot/certbot/pull/7360.